R/log.R

```diff
@@ -11,11 +11,19 @@
 log_push <- function(x, .f, path = proj_root()) {
   log <- log_touch(path)
   old_paths <- log_report(path)
+
+  # Get the absolute path of the files
+  if (is_file(x) | is_dir(x)){
+    abs = fs::path_abs(x)
+  } else{
+    abs = "N/A"
```

Comment:
Why not `NA`?

Reply:
I had been running into errors when using `NA`, but it turns out that was the result of another line of code being in the wrong spot. Just fixed that and switched everything over to `NA`.